model/include/model/ais_defs.h

```diff
@@ -44,8 +45,8 @@ typedef enum AisError {
   AIS_INCOMPLETE_MULTIPART
 } _AisError;
 
-WX_DECLARE_HASH_MAP(int, wxString, wxIntegerHash, wxIntegerEqual,
-                    AIS_Target_Name_Hash);
+WX_DECLARE_HASH_MAP(int, AisTargetCacheData, wxIntegerHash, wxIntegerEqual,
+                    AIS_Target_Data_Hash);
```

Comment:
Would certainly be better to use a std::unordered_map here. See https://opencpn-manuals.github.io/main/opencpn-dev/coding-guidelines.html#_c_practices